src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/catalog_price_rule.yml

```diff
@@ -54,3 +54,13 @@ admin_catalog_price_rules_bulk_delete:
     _controller: 'PrestaShopBundle\Controller\Admin\Sell\Catalog\CatalogPriceRuleController::bulkDeleteAction'
     _legacy_controller: AdminSpecificPriceRule
 #  _legacy_link: AdminSpecificPriceRule:submitBulkdeletespecific_price_rule
+
+admin_catalog_price_rules_list_for_product:
+  path: /list-for-product/{productId}
+  methods: [ GET ]
+  defaults:
+    _controller: 'PrestaShopBundle\Controller\Admin\Sell\Catalog\CatalogPriceRuleController::listForProductAction'
+    _legacy_controller: AdminSpecificPriceRule
+    # _legacy_link: AdminSpecificPriceRule:submitBulkdeletespecific_price_rule
```

Comment:
Not sure this one had a legacy equivalent But even if there is one I don't think it's related to bulk delete action so maybe this one can be removed